conf/test_alphafold_download.config

```diff
@@ -0,0 +1,34 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/proteinfold -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+stubRun = true
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data to test alphafold2 analysis
+    mode            = 'alphafold2'
+    alphafold2_mode = 'standard'
+    input           = params.pipelines_testdata_base_path + 'proteinfold/testdata/samplesheet/v1.0/samplesheet.csv'
+}
+
+process {
+    withName: 'ARIA2|UNTAR|RUN_ALPHAFOLD2' {
+        container = 'biocontainers/gawk:5.1.0'
```

Comment:
This test are `stub` runs since the DBs are too big to perform a real test each time, and also the real container for run the pipeline weights several `Gb` that is why we just used a small container that allow us to run these tests.